### ISSUE TITLE
KAS+ZIL

### ISSUE THREAD

**Issue by dolgikhi** [todxx/teamredminer#704]
Hello. How can I make the memory frequency increase during ZIL mining? Thanks for the answer.

**Comment by CamJava007** [comment]
Increase your memory frequency in your overclock to whatever you'd like it to be, before you start mining. The Kaspa KHeavyHash Algo does not use the cards memory at all, so you can set it to whatever you want it to be for Zil and it will not affect your Kaspa Mining.

**Comment by dolgikhi** [comment]
If I increase the memory frequency, the consumption of the card becomes larger.

**Comment by CamJava007** [comment]
Yes... that's how it works. You have to choose. Do you want higher hashrate or lower consumption.

**Comment by dolgikhi** [comment]
bzminer and gminer can increase the memory frequency during ZIL mining, it's a pity that TRM can't do that.

**Comment by jctopus** [comment]
Как понизить частоту ядра у зилики при дуал майнинге кас+ зил без потерь хешрейта у каспы
